spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_1_spec.rb

```diff
@@ -1,787 +0,0 @@
-describe ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher do
```

Comment:
Why do we want to rename this file? I am not able to say what are the differences.

Reply:
I am not renaming it, I am deleting it. We now have a new test file that does not have all those allow_to_receive_and_return things, instead it uses the OvirtSDK::VCR

Reply:
OK